tox.ini

```diff
@@ -39,7 +39,8 @@ commands =
 
 [testenv:type-checking]
 deps =
-    mypy
+    mypy>=0.901
+    types-ujson
```

Comment:
what's this for?

Reply:
It's for mypy, it won't pass without it

Reply:
Yeah, I just saw this on another pipeline for another project:

```
[2021-06-08T20:39:34.212Z] dolly/src/pf/dolly/workflow/runner.py:8: error: Library stubs not installed for "ujson" (or incompatible with Python 3.8)

[2021-06-08T20:39:34.212Z] dolly/src/pf/dolly/workflow/runner.py:8: note: Hint: "python3 -m pip install types-ujson"
```